File: JavaMavenProject/src/main/java/adamatti/FizzBuzz.java
package adamatti;

public class FizzBuzz {
    static String fizzBuzz(int number){
        StringBuffer sb = new StringBuffer();

        if (number % 3 == 0) sb.append("Fizz");
        if (number % 5 == 0) sb.append("Buzz");
        if (sb.length() == 0) sb.append(number);

        return sb.toString();
    }
}

File: TypeScriptProject/src/fizzBuzz.ts
export default function fizzBuzz(x:number){
    let result:string = ""
    if (x % 3 == 0) result+="Fizz"
    if (x % 5 == 0) result+="Buzz"
    if(!result)     result = `${x}`
    return result
}
